sql/mozfun/iap/scrub_apple_receipt/udf.sql

```diff
@@ -11,12 +11,12 @@ RETURNS STRUCT<
         SELECT
           TIMESTAMP_MILLIS(purchase_date_ms) AS start_time,
           TIMESTAMP_MILLIS(expires_date_ms) AS end_time,
-          iap.derive_apple_subscription_interval(
+          mozfun.iap.derive_apple_subscription_interval(
             DATETIME(TIMESTAMP_MILLIS(purchase_date_ms), "America/Los_Angeles"),
             DATETIME(TIMESTAMP_MILLIS(expires_date_ms), "America/Los_Angeles")
           ) AS `interval`,
         FROM
-          UNNEST(apple_receipt.receipt.in_app)
+          UNNEST(apple_receipt.latest_receipt_info)
```

Comment:
[per the documentation](https://developer.apple.com/documentation/appstorereceipts/responsebody), `latest_receipt_info` is all of the transactions, where `receipt` is just the one used for verification:

> Properties | _
> ---|---
> ... | ...
> latest_receipt_info [[responseBody.Latest_receipt_info](https://developer.apple.com/documentation/appstorereceipts/responsebody/latest_receipt_info)] | An array that contains all in-app purchase transactions. This excludes transactions for consumable products that have been marked as finished by your app. Only returned for receipts that contain auto-renewable subscriptions.
> ... | ...
> receipt [responseBody.Receipt](https://developer.apple.com/documentation/appstorereceipts/responsebody/receipt) | A JSON representation of the receipt that was sent for verification.